.babelrc

```diff
@@ -1,8 +1,3 @@
 {
-  "presets": ["es2015", "stage-0", "react"],
-  "ignore": [
-    "src/**/__snapshots__",
-    "src/**/*.test.js",
-    "src/**/*.stories.js"
-  ]
-}
+  "presets": ["es2015", "stage-0", "react"]
```

Comment:
I believe this PR should resolve the problem of running tests with travis: https://github.com/patternfly/patternfly-react/pull/16